backend/fregepoc/indexers/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from fregepoc.indexers.models import GitHubIndexer
+
+
+class GitHubIndexerSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = GitHubIndexer
+        fields = ['min_forks', 'min_stars', 'current_page', 'rate_limit_timeout', ]
```

Comment:
We don't intend to expose parts of the underlying logic - in this case the custom Indexer models through the API. Their primary use is to configure indexing behaviour, not be accessible externally.